### PR TITLE
[11.0][MIG] onchange_helper

### PR DESCRIPTION
**Pull request by astirpe**

### CODE REVIEW

File: onchange_helper/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Copyright 2016-2017 Akretion (http://www.akretion.com)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+{'name': 'Onchange Helper',
+ 'version': '11.0.1.0.0',
+ 'summary': 'Technical module that ease execution of onchange in Python code',
+ 'author': 'Akretion,Camptocamp,Odoo Community Association (OCA)',
+ 'website': 'http://www.akretion.com',
```
Comment (simahawk) [inline onchange_helper/__manifest__.py]: https://github.com/OCA/server-tools

File: onchange_helper/models/ir_rule.py [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+                                              field, onchange_specs)
+        new_values.update(get_new_values(self, values, onchange_values))
+        all_values.update(new_values)
+
+    res = {f: v for f, v in all_values.items()
+           if f in values or f in new_values}
+    return res
+
+
+class IrRule(models.Model):
```
Comment (simahawk) [inline onchange_helper/models/ir_rule.py]: why don't we make `BaseModel` inherit to catch them all?

File: onchange_helper/README.rst [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+
+This is a technical module. Its goal is to ease the play of onchange method directly called from Python code.
+
+Usage
+=====
+
+To use this module, you need to:
+
+* depend on this module
+* call `yourmodel.play_onchanges(values, ['field'])`
```
Comment (simahawk) [inline onchange_helper/README.rst:17]: this not correct actually, as the only model considered by this module is `ir.rule` (see my comment below)

**Review by simahawk — CHANGES_REQUESTED** [review]

File: onchange_helper/models/ir_rule.py (hunk 2 above)
Comment (astirpe) [reply onchange_helper/models/ir_rule.py]: Not sure why it was like that. Let's see how behaves with BaseModel.

File: onchange_helper/README.rst (hunk 3 above)
Comment (astirpe) [reply onchange_helper/README.rst:17]: Should be fixed now, thank you!

**Comment by astirpe** [comment]
Squashed

**Review by simahawk — APPROVED** [review]
tnx for adding tests ;)

**Comment by simahawk** [comment]
@guewen @florian-dacosta ping :)

**Review by etobella — APPROVED** [review]
LGTM :+1:

**Review by rgarnau — APPROVED** [review]
LGTM :+1:

**Review by JordiBForgeFlow — APPROVED** [review]
:+1: Technical and functional review. Very useful!!

**Pull request merged**